docs/prd/026-declarative-web-push.md

```
@@ -0,0 +1,479 @@
+    "silent": false,
+    "data": {
+      "category": "job_failure",
+      "notification_guid": "ntf_01hgw..."
+    }
+  },
+  "app_badge": 1
+}
+```
+
+**Required fields:**
+| Field | Type | Description |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)